bin/windows/kafka-consumer-perf-test.bat

```diff
@@ -14,7 +14,11 @@ rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 rem See the License for the specific language governing permissions and
 rem limitations under the License.
 
+REM ***
+REM This is a modified version of script that works correctly with paths containing spaces.
+REM ***
```

Comment:
I don't think this comment is needed here, since it would be meaningless after the PR is merged.